### ISSUE TITLE
Newrelic recipe triggers a Symfony style error

### ISSUE THREAD

**Issue by mjrider** [deployphp/recipes#114]
the code on line https://github.com/deployphp/recipes/blob/master/newrelic.php#L34
Triggers the following error:
```
 [ERROR] Symfony\Component\Console\Exception\InvalidArgumentException: Incorrectly nested style tag found. in        
         /home/muller/.config/composer/vendor/symfony/console/Formatter/OutputFormatterStyleStack.php:87             
         Stack trace:                                                                                                
         #0 /home/muller/.config/composer/vendor/symfony/console/Formatter/OutputFormatter.php(185):                 
         Symfony\Component\Console\Formatter\OutputFormatterStyleStack->pop(Object(Symfony\Component\Console\Formatte
         r\OutputFormatterStyle))                                                                                    
         #1 /home/muller/.config/composer/vendor/symfony/console/Application.php(653):                               
         Symfony\Component\Console\Formatter\OutputFormatter->format('<error>  d'</in...')                           
         #2 /home/muller/.config/composer/vendor/symfony/console/Application.php(127):                               
         Symfony\Component\Console\Application->renderException(Object(RuntimeException),                            
         Object(Symfony\Component\Console\Output\StreamOutput))                                                      
         #3 /home/muller/.config/composer/vendor/deployer/deployer/src/Deployer.php(190):                            
         Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput),               
         Object(Symfony\Component\Console\Output\ConsoleOutput))                                                     
         #4 /home/muller/.config/composer/vendor/deployer/deployer/bin/dep(120): Deployer\Deployer->run()            
         #5 {main} 
```

This seems triggerd by the '=>' after application_id

**Comment by elfet** [comment]
Can you send PR with fix?

**Comment by mjrider** [comment]
oke, some more debugging, i think it is maybe not related to deployer
it only breaks when your terminal is between  72 and 134 columns

**Comment by elfet** [comment]
Maybe delete `=>` at all?

```
<info>set('newrelic', ['api_key', 'application_id']);</info>
```

**Comment by mjrider** [comment]
yes that would work around the problem.

i think that we just found a bug in symfony, but my symfony knowledge is to  small to create a small reproduction script so i could file a bug with symfony

**Comment by elfet** [comment]
I think, lets fix this recipe first. Maybe someone get this too.

**Comment by mjrider** [comment]
oke, made a reproduction and a bugreport
https://github.com/symfony/symfony/issues/22021
https://github.com/mjrider/break-symfony-console

**Comment by jordscream** [comment]
@mjrider , I have this in local

<img width="702" alt="capture d ecran 2017-03-16 a 14 18 26" src="https://cloud.githubusercontent.com/assets/616999/23997914/946d1e12-0a53-11e7-86f9-aebd0dd7f30e.png">

**Comment by mjrider** [comment]
@jordscream did you try it on different terminal sizes ? because if the terminal is small or big enough, it will not fail

**Comment by jordscream** [comment]
@mjrider , Indeed I reproduce the problem after 74 columns :-/

**Comment by mjrider** [comment]
![screenshot at 2017-03-16 14-38-48](https://cloud.githubusercontent.com/assets/213105/23998753/5c28b59a-0a56-11e7-89e8-a87c847fd85b.png)
see the broken `</info>` tag in the output and the tput cols output

**Comment by jordscream** [comment]
@mjrider it's effectly a symfony bug. I'll check

**Comment by mjrider** [comment]
Looks like symfony is going for the option of removing formatting from exceptions.
https://github.com/symfony/symfony/pull/22142

**Comment by chalasr** [comment]
In fact, the exception message will stay raw, for instance `<info>Exception!</info>` will stay as is in the output. I suggest you to don't throw the exception, but use the SymfonyStyle API instead (see [`SymfonyStyle::block()`](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Style/SymfonyStyle.php#L68), [`SymfonyStyle::error()`](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Style/SymfonyStyle.php#L151)), returning an appropriated error code. The patch has been merged in Symfony from 2.7 up to master and will be part of the next maintenance releases.
